src/main/java/org/jabref/gui/edit/CopyMoreAction.java

```diff
@@ -113,6 +114,33 @@ private void copyKey() {
         }
     }
 
+    private void copyDOI() {
+        List<BibEntry> entries = stateManager.getSelectedEntries();
+
+        // Collect all non-null DOIs.
+        List<String> DOIs = entries.stream()
```

Comment:
variables should be all lower camel case, event is they are abbreviations => doi

Reply:
This should be addressed by commit 595945eb3c918f330f00d4b604382fe32fbb1f66. Thanks for the feedback.